src/main/drivers/serial_uart.h

```diff
@@ -41,7 +41,9 @@ typedef enum {
     UARTDEV_9 = 8,
     UARTDEV_10 = 9,
     LPUARTDEV_1 = 10,
+    UARTDEV_COUNT = 11
```

Comment:
```suggestion
    UARTDEV_COUNT
```
I'd suggest to just leave `UARTDEV_COUNT` without assigning a distinct number, so that for future additions to the enum we don't need to touch this line and `UARTDEV_COUNT` gets incremented automatically.

Reply:
Resolved.

Reply:
@KarateBrot - I'm working on it - if you want to take over no problem. The PR was stalled.

Have rebased the PR and added a new commit  - after review request from @SteveCEvans